bigbio/hub/hub_repos/symptemist/README.md

```diff
@@ -0,0 +1,39 @@
+---
+language:
+  - es
+bigbio_language:
+  - Spanish
+license: Creative Commons Attribution 4.0 International License
+bigbio_license_shortname: CC_BY_4p0
+multilinguality: monolingual
+pretty_name: SympTEMIST
+homepage: https://temu.bsc.es/symptemist/
+bigbio_pubmed: false
+bigbio_public: true
+bigbio_tasks:
+  - NAMED_ENTITY_RECOGNITION
+  - NAMED_ENTITY_DISAMBIGUATION
+---
+
+
+# Dataset Card for SympTEMIST
+
+## Dataset Description
+
+- **Homepage:** https://temu.bsc.es/symptemist/
```

Comment:
The entity and document statistics reported on the webpage (https://temu.bsc.es/symptemist/corpus-description/) do not match those reported by the tests (see below). Is this expected?

```
train
==========
id: 304
document_id: 304
passages: 304
entities: 3484
normalized: 3484
events: 0
coreferences: 0
relations: 0

test
==========
id: 246
document_id: 246
passages: 246
entities: 2848
normalized: 2848
events: 0
coreferences: 0
relations: 0
```